src/tfvc/tfvcscmprovider.ts

```diff
@@ -173,4 +173,33 @@ export class TfvcSCMProvider implements SCMProvider {
 
         return "";
     }
+
+    private static ResolveTfvcURI(uri: Uri): SCMResource | SCMResourceGroup | undefined {
+        if (uri.authority !== TfvcSCMProvider.scmScheme) {
+            return;
+        }
+
+        return scm.getResourceFromURI(uri);
+    }
+
+    private static ResolveTfvcResource(uri: Uri): Resource | undefined {
```

Comment:
personally, I don't think we should include the "| undefined". It seems unnecessary.